src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/reference/Cas2PersistedApplicationStatus.kt

```diff
@@ -7,5 +7,6 @@ data class Cas2PersistedApplicationStatus(
   val name: String,
   val label: String,
   val description: String,
+  val statusDetails: List<Cas2PersistedApplicationStatusDetail>? = null,
```

Comment:
would it be possible to make this non-nullable and initialise to emptyList()? Doesn't matter too much in kotlin as we have nice nullable control but can make calling code a bit simpler maybe?